webpack.config.dev.js

```diff
@@ -0,0 +1,71 @@
+// We are using node's native package 'path'
```

Comment:
consider putting this in a config/ directory

Reply:
eventually, the config/ directory will contain
- webpack.config.dev.js
- webpack.config.prod.js
- webpack.config.base.js (or some similar name)